### ISSUE TITLE
[DISCUSSION] TSDoc

### ISSUE THREAD

**Issue by MattMattV** [compodoc/compodoc#634]
I just found the [TSDoc](https://github.com/Microsoft/tsdoc) repo from Microsoft, that tries to unify efforts to improve state of TypeScript documentation, to have all the TS developers use the same "documentation syntax spec"

If this project become stable one day it could help compodoc

I hope this information will be useful !

**Comment by vogloblinsky** [comment]
Yes thanks for creating this thread. I heard about this new library which could be very useful.
There are related issues on JSDoc parsing : https://github.com/compodoc/compodoc/labels/Context%20%3A%20JSDoc
I will give a try with one of these issues, and if it is positive, switching to it.

**Comment by arjunyel** [comment]
Im about to start trying compodoc, @MattMattV does this owrk with TSDoc out of the box? Thanks for the help!

**Comment by vogloblinsky** [comment]
@arjunyel Compodoc doesn't work out of the box with TSDoc, it use a custom implementation linked with TypeScript internal JSDoc parsing.
TSDoc is a new parser for JSDoc comments with TypeScript with seems more robust.
As i said earlier, i will give a try soon.

**Comment by vogloblinsky** [comment]
Just started testing tsdoc. @example support which is the main issue in Compodoc is not implemented right now : https://github.com/Microsoft/tsdoc/issues/20

**Comment by vogloblinsky** [comment]
[] - have a look at https://github.com/Microsoft/web-build-tools/tree/master/apps/api-extractor
